src/Service.Host/client/src/components/StorageTypes.js

```diff
@@ -0,0 +1,217 @@
+import React, { useEffect, useState } from 'react';
+import Typography from '@mui/material/Typography';
+import List from '@mui/material/List';
+import Grid from '@mui/material/Grid2';
+import { DataGrid } from '@mui/x-data-grid';
+import { Loading, ErrorCard, SnackbarMessage } from '@linn-it/linn-form-components-library';
+import Button from '@mui/material/Button';
+import Page from './Page';
+import config from '../config';
+import itemTypes from '../itemTypes';
+import usePut from '../hooks/usePut';
+import useInitialise from '../hooks/useInitialise';
+import usePost from '../hooks/usePost';
+import useGet from '../hooks/useGet';
+
+function StorageTypes() {
+    const { isLoading, result: storageTypesResult } = useInitialise(itemTypes.storageTypes.url);
```

Comment:
I think I'd just ditch the useInitialise hook and solely make use of useGet so you dont need two different sets of variables.